v2/inc/c_logging/log_context.h

```diff
@@ -47,6 +47,7 @@ const LOG_CONTEXT_PROPERTY_VALUE_PAIR* log_context_get_property_value_pairs(LOG_
 int internal_log_context_init_from_parent(LOG_CONTEXT_HANDLE dest_log_context, LOG_CONTEXT_HANDLE parent_log_context);
 uint32_t internal_log_context_get_values_data_length_or_zero(LOG_CONTEXT_HANDLE log_context);
 
+MU_SUPPRESS_WARNING(4505) // warning C4505: 'internal_log_context_get_property_value_pair_count_or_zero': unreferenced function with internal linkage has been removed
 static uint32_t internal_log_context_get_property_value_pair_count_or_zero(LOG_CONTEXT_HANDLE log_context)
```

Comment:
>internal_log_context_get_property_value_pair_count_or_zero [](http://example.com/codeflow?start=16&length=58)

this whole function seems to exist because it wants to switch the result of `log_context_get_property_value_pair_count` (note: public API) from UINT32_MAX to 0. 

What would happen : 
if "1" :`log_context_get_property_value_pair_count` would return "0" when it returns UINT32_MAX?

if "2": whoever is calling now `internal_log_context_get_property_value_pair_count_or_zero` sould actually call `log_context_get_property_value_pair_count` and examine the return value itself?

Reply:
So, did this one. Makes everything simpler.